org-formation/organization.yaml

```diff
@@ -356,8 +356,6 @@ Organization:
         <<: !Include ./_default_org_tags.yaml
         Department: IBC
         Project: Infrastructure
-        budget-alarm-threshold: 3000
```

Comment:
could you please add a comment referencing the more granular budget alerts and where it's done?

Reply:
Done!